chpl/chpl-service/src/main/java/gov/healthit/chpl/attestation/entity/AttestationPeriodDeveloperExceptionEntity.java

```diff
@@ -0,0 +1,61 @@
+package gov.healthit.chpl.attestation.entity;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import gov.healthit.chpl.entity.developer.DeveloperEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
```

Comment:
I know there are thoughts about Lombok annotations, but `@Data` would eliminate these three, and add `@EqualsAndHashCode` which I think we end up needing sometimes. Is it worth talking about making a choice betwene either `@Data` and no specific ones, or eliminating the various "combination" annotations from Lombok, including `@Data`?